docs/docs/docs_es/introduction/core-concepts.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Conceptos Principales
+---
+
+## Visión General
+
+Recoil te permite crear flujos de datos con grafos que fluyen desde los _átomos_ (el estado compartido), pasando por los _selectores_ (funciones puras) y finalmente llegando a tus componentes de React. Los átomos son unidades de estado, a los cuales los componentes se pueden subscribir. Y los selectores transforman este estado de manera síncrona o asíncrona
```

Comment:
Remove te, los átomos->átomos, same for selectores and tus before componentes. A los que cada componente se puede suscribir. Remove Y